content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -114,3 +116,151 @@ The scheduler (through the _VolumeZonePredicate_ predicate) also will ensure tha
+
+Used on: Service
+
+The label is the tag name we use to differentiate multiple services. Used currently for ELBs only.
+
+See [kubernetes.io/service-name](#kubernetesioservice-name).
+
+## endpointslice.kubernetes.io/managed-by {#endpointslicekubernetesiomanaged-by}
+
+Example: `endpointslice.kubernetes.io/managed-by="controller"`
+
+Used on: Endpoint
+
+The label is used to indicate the controller or entity that manages an EndpointSlice. This label aims to enable different EndpointSlice objects to be managed by different controllers or entities within the same cluster.
+
+When upgrading to v1.17 with a cluster with EndpointSlice enabled, the `endpointslice.kubernetes.io/managed-by` label needs to be set on each EndpointSlice.
+
+See [endpointslice.kubernetes.io/managed-by](#endpointslicekubernetesiomanaged-by).
+
+## endpointslice.kubernetes.io/skip-mirror {#endpointslicekubernetesioskip-mirror}
+
+Example: `endpointslice.kubernetes.io/skip-mirror="true"`
+
+Used on: Endpoint
+
+The label can be set to true on an Endpoint resource to indicate that the EndpointSliceMirroring controller should not mirror this resource with EndpointSlices.
+
+See [endpointslice.kubernetes.io/skip-mirror](#endpointslicekubernetesioskip-mirror).
+
+## service.kubernetes.io/service-proxy-name {#servicekubernetesioservice-proxy-name}
+
+Example: `service.kubernetes.io/service-proxy-name="foo-bar"`
+
+Used on: Service
+
+The label indicates that a custom service proxy will implement this service, kube-proxy does not generate iptables for the service.
+
+See [service.kubernetes.io/service-proxy-name](#servicekubernetesioservice-proxy-name).
+
+# Annotations
+
+## experimental.windows.kubernetes.io/isolation-type
+
+Example: `experimental.windows.kubernetes.io/isolation-type: "hyperv"`
+
+The annotation is used to run windows containers with hyperv isolation.
+
+At the moment this annotation only supports one container per pod.
+
+## ingressclass.kubernetes.io/is-default-class
+
+Example: `ingressclass.kubernetes.io/is-default-class: "true"`
+
+When a single IngressClass resource has this annotation set to true, new Ingress resource without a class specified will be assigned this default class.
+
+## kubernetes.io/ingress.class (deprecated)
+
+{{< note >}} Starting in v1.18, this annotation is deprecated in favor of [spec.ingressClassName]. {{< /note >}}
+
+## alpha.kubernetes.io/provided-node-ip
+
+Example: `alpha.kubernetes.io/provided-node-ip: "10.0.0.1"`
+
+The annotation is a node IP annotation set by the "external" cloud provider.
+
+When kubelet is started with the "external" cloud provider, then it sets this annotation on the node to denote an ip address set from the cmd line flag (node-ip). This ip is verified with the cloudprovider as valid by the cloud-controller-manager.
+
+# Taints
+
+## node.kubernetes.io/not-ready
+
+Example: `node.kubernetes.io/not-ready:NoExecute`
+
+The taint will be added when node is not ready and removed when node becomes ready.
```

Comment:
Maybe add "The Node controller monitors the health of all Nodes and adds or removes this taint accordingly."

Reply:
Could link to https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-nodes-by-condition

My opinion: it's OK for this page to say “the control plane” rather than “the Node controller”; specific details about how it's implemented would belong elsewhere.